server/src/main/scala/controllers/ApplicationWithImports.scala

```diff
@@ -0,0 +1,36 @@
+package controllers
+
+import java.nio.file.Paths
+import java.util.UUID
+
+import drt.server.feeds.lhr.LHRForecastFeed
+import drt.server.feeds.lhr.forecast.LHRForecastXLSExtractor
+import drt.shared.FlightsApi.Flights
+import play.api.libs.Files
+import play.api.mvc.{Action, Request}
+import server.feeds.StoreFeedImportArrivals
+
+import scala.language.postfixOps
+import scala.util.Success
+
+trait ApplicationWithImports {
+  self: Application =>
+
+  def feedImport(feedType: String, portCode: String): Action[Files.TemporaryFile] = Action(parse.temporaryFile) { request: Request[Files.TemporaryFile] =>
+    val filePath = s"/tmp/${UUID.randomUUID().toString}"
+
+    request.body.moveTo(Paths.get(filePath), replace = true)
```

Comment:
I think we should maybe do some sort of checking on this file - I believe that ACP has a build in virus scanner thing we might be able to use.